course/models.py

```diff
@@ -450,6 +450,11 @@ def __str__(self):
     def clean(self):
         super().clean()
         errors = {}
+        RESERVED = ("instances",)
+        if self.instance_name in RESERVED:
+            errors['instance_name'] = _("You cannot use word '{}' as an instance name.").format(self.instance_name)
```

Comment:
I realized now that lazy strings must not be formatted with the normal string `.format()` method. Django has the function `format_lazy()` for lazy strings. In this module, `_` refers to `gettext_lazy`. 

https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#formatting-strings-format-lazy
https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.text.format_lazy
https://github.com/django/django/blob/stable/2.2.x/django/utils/text.py#L408
https://github.com/django/django/blob/stable/2.2.x/django/utils/functional.py#L92